RadClass/RadClass.py

```diff
@@ -86,23 +92,38 @@ def queue_file(self):
 
         self.processor = ds.DataSet(self.labels)
         self.processor.init_database(self.filename, self.datapath)
-        # parameters for keeping track of progress through a file
 
+        # parameters for keeping track of progress through a file
         self.start_i = 0
         self.stop_i = len(self.processor.timestamps) - 1
 
         if self.start_time is not None:
             timestamp = self.processor.timestamps[self.processor.timestamps >=
-                                                  self.start_time][0]
+                                                  self.start_time]
+            # check if any timestamps were found
+            if timestamp.size == 0:
+                raise ValueError('User start time is larger than all timestamps.')
```

Comment:
Note that Coveralls coverage decreased mainly due to the ValueError lines. Do I want to create pytests that deliberately raise these errors?

Reply:
That would be appropriate - you want to know that it actually does raise an error when it's supposed tod.

Reply:
Strange...